src/plugins/projectGalaxy/plugin.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Project Galaxy",
+  "version": "0.0.1",
+  "author": "Project-Galaxy",
+  "description": "Project Galaxy OAT",
+  "website": "https://github.com/snapshot-labs/snapshot/tree/develop/src/plugins/projectGalaxy",
+  "icon": "https://ipfs.io/ipfs/bafkreibcmyqzyltbo5flwdrezyadm7zi66ka2oidwa22g4g7z6t6r4v4ui"
```

Comment:
The new readme is good but I think I'd still put defaults here to prefill the settings form.

```suggestion
  "icon": "https://ipfs.io/ipfs/bafkreibcmyqzyltbo5flwdrezyadm7zi66ka2oidwa22g4g7z6t6r4v4ui",
  "defaults": {
    "space": {
      "oats": {
        "<Proposal ID>": "<Space Name>/campaign/<Campaign ID>"
      }
    }
  }
```

![image](https://user-images.githubusercontent.com/6792578/176151792-53ecfae5-34af-43dd-8df2-4940c83aca6e.png)

I guess I didn't mention that this will prefill the form, which is of course more convenient than looking things up in a readme.

Btw... is this string in the middle `/campaign/` really needed? Can't it just be `<Space Name>/<Campaign ID>` ?

Reply:
*/campaign/* part is designed by the backend coworkers, and i think its necessary for different purposes, like *<Space Name>/credential/<Credential ID>*